pipeline/relation_embedder/relation_embedder/src/main/scala/uk/ac/wellcome/relation_embedder/RelationEmbedderWorkerService.scala

```diff
@@ -53,7 +53,12 @@ class RelationEmbedderWorkerService[MsgDestination](
             val denormalisedWorks = relationsService
               .getAffectedWorks(batch)
               .map { work =>
-                work.transition[Denormalised](relationsCache(work))
+                val relations = relationsCache(work)
+                val relationAvailabilities =
+                  relationsCache.getAvailabilities(work)
+                work.transition[Denormalised](
+                  (relations, relationAvailabilities)
+                )
```

Comment:
Not for this PR, but I'd quite like a way for us to support multiple arguments rather than tuples here, because I'll be doing the same for the redirect sources work.

Reply:
Very much agreed. Case classes would be an easy improvement too